client/verta/tests/conftest.py

```diff
@@ -36,6 +36,8 @@
     mock_env_jwt_auth,
     mock_env_authn_missing,
 )
+from verta._internal_utils._utils import generate_default_name
+from verta._protos.public.uac import RoleV2_pb2 as _Role
```

Comment:
See my comment in `_organization.py`

```suggestion
from verta._protos.public.uac import RoleV2_pb2
```